src/pages/home/report/ReportActionCompose/ComposerWithSuggestions.js

```diff
@@ -290,7 +291,10 @@ function ComposerWithSuggestions({
 
                 const parentReportActionID = lodashGet(report, 'parentReportActionID', '');
                 const parentReportAction = lodashGet(parentReportActions, [parentReportActionID], {});
-                const lastReportAction = _.find([...reportActions, parentReportAction], (action) => ReportUtils.canEditReportAction(action));
+                const lastReportAction = _.find(
+                    [...reportActions, parentReportAction],
+                    (action) => ReportUtils.canEditReportAction(action) && !ReportActionsUtils.isMoneyRequestPreviewAction(action),
```

Comment:
ReportActionsUtils.isMoneyRequestPreviewAction  - Why condition is that too complex? 
Any problem if just using `ReportUtils.canEditReportAction(action) && ReportActionsUtils .isMoneyRequestAction(action)`?
System message is also not editable

Reply:
@hungvu193 this is deploy blocker so we should fix asap. Are you available now to handle my feedback?

Reply:
yeah I think I agree here, can probably undo most of the changes and just add a check of `&& ReportActionsUtils.isMoneyRequestAction(action)` here and that should do it?

Reply:
+1

Reply:
@luacmartins are you planning to deploy today?
If so, I will create new PR with simple fix as @hungvu193 won't be available next few hrs based on their timezone

Reply:
@situchan yes, that's my plan

Reply:
I think we can close this PR. We already merged a fix.